rclpy/rclpy/executors.py

```diff
@@ -105,6 +120,22 @@ def __init__(self):
         self._last_args = None
         self._last_kwargs = None
 
+    def create_task(self, callback, *args):
+        """
+        Add a callback or coroutine to be executed during :meth:`spin` and return a Future.
+
+        Arguments to this function are passed to the callback.
```

Comment:
Could this also permit keyword arguments?

Reply:
Yup, Added in 5f2cb1eb6cc451acbe0bb58b93672ebe224b45e6

Reply:
I didn't think about it before. It might be necessary to pass `executor` using a setter function instead to avoid a collision if the callback wants to use a keyword argument with that name.

Reply:
In 5f2cb1e the expansion of `args` and `kwargs` in `Task.__init__()` [was removed](https://github.com/ros2/rclpy/commit/5f2cb1eb6cc451acbe0bb58b93672ebe224b45e6#diff-521ddf9ec814122daa49f03bb2974340R133) to prevent that collision. The expansion on `executor.create_task()` cannot conflict because they're now passed to `Task.__init__()` as a sequence and dictionary.

Reply:
Perfect!